core/templates/dev/head/app.js

```diff
@@ -71,6 +71,15 @@ oppia.constant(
   'RTE_COMPONENT_SPECS',
   window.GLOBALS.RTE_COMPONENT_SPECS ? window.GLOBALS.RTE_COMPONENT_SPECS : {});
 
+oppia.constant(
+  'AUDIO_UPLOAD_URL_TEMPLATE',
+  '/createhandler/audioupload/<exploration_id>');
+
+oppia.constant(
+  'GCS_AUDIO_DOWNLOAD_URL_TEMPLATE',
+  'https://storage.googleapis.com/oppiatestserver-resources/' +
```

Comment:
The bucket name should be read from constants.js.

I guess that means this isn't a constant anymore though. Does it actually need to be a global constant or can it be constructed internally within a service instead?